sandbox/grommet-app/src/pages/layouts/kinds/DSCCSystemDetail/PhysicalCapacity.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { Box, Grid, Meter, Text } from "grommet";
+import { DashboardCard, Metric } from "../../../../components";
+
+
+export const PhysicalCapacity = ({ size, ...rest }) => {
+  return (
+    <DashboardCard
+      title="Physical capacity"
+      subtitle={null}
+      level={3}
+      hideCta
+      icon={null}
+      external={null}
+      footer={null}
+      inline={null}
+      onClick={null}
+      {...rest}
+    >
+      <Grid
+        gap="small"
+        areas={[
+          ["metric", "summary"],
+          ["meter", "meter"]
+        ]}
+        columns={['flex', 'flex']}
+        rows={['auto']}
+      >
+        <Metric
+          gridArea="metric"
+          label={null}
+          value="47%"
+          unit="used"
+          options={null}
+          reverse
+          size={size}
+        />
+        <Meter
```

Comment:
<img width="496" alt="Screenshot 2025-02-20 at 10 50 23 AM" src="https://github.com/user-attachments/assets/0bafb064-8764-4df2-9105-0dbd03c01897" />

This might be the nature of the existing design, but I'm find it hard to discern what the different colors mean. Would a legend be helpful?

Reply:
This the treatment from the reference design. Trying to not override the design decisions too much, more so just so we can demonstrate how they translate into the new theme. We can refine if there are layouts we want to promote and incorporate best practices at that time.